connectors/v2/dreamfm.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+/**
+ * remove zero width characters & trim
+ * @param  {string} text to clean up
+ * @return {string} cleaned up text
+ */
+function cleanText(input) {
```

Comment:
We should probably put this in the base connector now that two extensions are using similar functionality.